src/test/java/hudson/plugins/promoted_builds/KeepBuildForeverActionTest.java

```diff
@@ -50,6 +56,11 @@ public void testDoesNotMarkBuildIfPromotionNotGoodEnough() throws Exception {
         promotionJob.getBuildSteps().add(new FixedResultBuilder(Result.FAILURE));
         promotionJob.getBuildSteps().add(new KeepBuildForeverAction());
 
+        // Make this test work with Jenkins 1.575+
+        for (ItemListener l : ItemListener.all()) {
```

Comment:
@oleg-nenashev I took the idea from this comment https://github.com/jenkinsci/promoted-builds-plugin/blob/master/src/test/java/hudson/plugins/promoted_builds/PromotionProcessTest.java#L56 it seemed more suitable than using @LocalData. But any ideas/complaints are more than welcome